src/software/pipeline/main_panoramaCompositing.cpp

```diff
@@ -1005,7 +2081,7 @@ class LaplacianCompositer : public Compositer
     minsize / 8 = 2^x
     x = log2(minsize/8)
     */
-    size_t optimal_scale = size_t(floor(std::log2(double(minsize) / 8.0)));
+    size_t optimal_scale = size_t(floor(std::log2(double(minsize) / 5.0)));
```

Comment:
Could you explain in the comments where the `5.0` means?
I would recommend to create a dedicated const variable, so we can have an explicit name for the constants.

Reply:
Will do it